s3/sc-s3-encrypted-ra.yaml

```diff
@@ -25,8 +26,19 @@ Parameters:
     Type: String
     Description: (Optional) Name of the created bucket.
     Default: ""
+  RestrictSameRegionDownload:
+    Type: String
+    Description: >
+      (Optional) Data transfers within the same region between AWS resources are free.
+      true to restrict downloading files from this bucket to only AWS resources (e.g. EC2 , Lambda) within the same region as this bucket.
+      This will not allow even the owner of the bucket to download objects in this bucket when not using an AWS resource in the same region!
+    AllowedValues:
+      - true
+      - false
+    Default: true
```

Comment:
hmm, wondering whether this should be default to `false` instead?

Reply:
I left it as true because the `aws-infra` had it default to true. I can change it.